src/azure-cli/azure/cli/command_modules/appservice/resources/LinuxFunctionsStacks.json

```diff
@@ -0,0 +1,245 @@
+{
+    "value": [
+        {
+            "id": null,
+            "name": "dotnet",
+            "type": "Microsoft.Web/availableStacks?osTypeSelected=LinuxFunctions",
+            "properties": {
+                "name": "dotnet",
+                "display": ".NET Core",
+                "dependency": null,
+                "majorVersions": [
+                    {
+                        "displayVersion": "3.1",
+                        "runtimeVersion": "dotnet|3.1",
+                        "supportedFunctionsExtensionVersions": [
+                            "~3"
+                        ],
+                        "isDefault": true,
+                        "minorVersions": [],
+                        "applicationInsights": true,
+                        "appSettingsDictionary": {
+                            "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+                        },
+                        "siteConfigPropertiesDictionary": {
+                            "Use32BitWorkerProcess": false,
+                            "linuxFxVersion": "dotnet|3.1"
+                        },
+                        "isPreview": false,
+                        "isDeprecated": false,
+                        "isHidden": false
+                    },
+                    {
+                        "displayVersion": "2.2",
+                        "runtimeVersion": "dotnet|2.2",
+                        "supportedFunctionsExtensionVersions": [
+                            "~2"
+                        ],
+                        "isDefault": true,
```

Comment:
Can you please link the issue here to add support of creating V1 apps?

Reply:
In https://github.com/Azure/Azure-Functions/issues/1635, we determined we're not supporting V1 apps. So we need to explicitly ignore V1 apps with this workitem: https://github.com/Azure/Azure-Functions/issues/1637